core/src/main/resources/META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/native-image.properties

```diff
@@ -1,3 +1,8 @@
 Args =\
+  -H:+UnlockExperimentalVMOptions \
   -H:+AddAllCharsets \
+  -H:+BuildReport \
+  -H:ReflectionConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/jni-unix-socket-config.json \
+  -H:ResourceConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/resource-config.json \
+  -H:DynamicProxyConfigurationResources=META-INF/native-image/com.google.cloud.sql/cloud-sql-jdbc-socket-factory-parent/proxy-config.json \
```

Comment:
optional q: I wonder if we need to specify these JSONs in native-image.properties? The compiler automatically picks up config files under the `META-INF/native-image` directory if the files follow a standard naming pattern ([link](https://www.graalvm.org/latest/reference-manual/native-image/overview/BuildConfiguration/#embed-a-configuration-file))